src/components/Tasks/Sidebar.tsx

```diff
@@ -61,12 +80,16 @@ export default React.memo(function Sidebar(props: { tasks: TaskType[] }) {
   ));
 
   return (
-    <List dense>
+    <List>
       <SidebarListItem name={null} primaryText="All" icon={<InboxIcon />} amount={tasks.length} />
       <SidebarListItem name="today" primaryText="Due today" icon={<TodayIcon />} amount={amountDueToday} />
 
       <ListSubheader>Tags</ListSubheader>
       {tagsList}
+
+      <Divider style={{ marginTop: '1em', marginBottom: '1em' }} />
```

Comment:
You can just use 'marginVertical'

Reply:
oh cool I had no idea. Is that a React feature? or MUI?

Reply:
Please ignore me, it's a react-native thing, not a reactjs thing.